parm/ocnicepost/ice.csv

```diff
@@ -1,8 +1,11 @@
 ! variable name, dimension, grid location, remapping method, vector pair, grid location of vector pair
-  'hi_h', 2 , 'Ct', 'bilinear', '', ''
-  'hs_h', 2 , 'Ct', 'bilinear', '', ''
-'aice_h', 2 , 'Ct', 'bilinear', '', ''
- 'sst_h', 2 , 'Ct', 'bilinear', '', ''
```

Comment:
@GwenChen-NOAA 
Is there a reason not to retain this?  I understand the WMO product does not need this variable, but I can imagine it useful for some of the model development team to look at this field directly from the intermediate interpolated netCDF file.
One could not include this variable when writing out the grib2 product file.
Thoughts?

Reply:
`sst_h` is also outputted in the ocean files as `SST`. We had a discussion on duplicate variables across different components when developing the [document](https://docs.google.com/document/d/1AwYbHArLJ6KvD0HW5bUbNOiFJaVAWgWEmnMQ2g076TA/edit), and the conclusion is to only output duplicate variables in its originated model/component. For SST, it is originated from MOM6, so it will only output in ocean files. The sets of variables specified in the [document](https://docs.google.com/document/d/1AwYbHArLJ6KvD0HW5bUbNOiFJaVAWgWEmnMQ2g076TA/edit) apply to both tripolar netCDF files and latlon GRIB2 files. If the interpolation algorithm in ocnicepost is used on the tripolar netCDF files, they will not contain `sst_h` to begin with.

Reply:
I understand that the variables will not be duplicated in the final product (grib2).  This intermediate netCDF file is not a product. Does having an interpolated `sst_h` in the intermediate ice interpolated netCDF gets mapped to the ice product file?  If so, that program should only do IO conversions on the variable it needs and not everything in the intermediate file. (This is my suggestion).
Please correct me if I am mistaken.

Reply:
We do need additional variables to be added to meet the requirements.

Reply:
Yes. I understand that.  My question was about removing variables not needed in the final product file.

Reply:
It depends on the steps you do to generate operational tripolar netCDF files.

If you first generate operational tripolar netCDF files from model outputs, then apply the interpolation algorithm, `sst_h` won't be in the operational netCDF files.

If you apply the interpolation directly to model output tripolar netCDF files and generate GRIB2 files from the intermediate interpolated netCDF files, then it doesn't matter what contain in the intermediate files. This approach requires operational tripolar netCDF files to be generated in a separate step.

Reply:
I'm confused by what you are referring to as "operational tripole netcdf" files. Both MOM6 and CICE6 only write tripole netCDF files for model output

Reply:
Ok. Let me clarify that then.
The model generates netCDF files on the native (tripolar) grid.  The contents of this model output netCDF are defined in a file named `diag_table` (for ocean).  For ice, I am not sure (@DeniseWorthen ??)
Regardless, the model will produce these fields on the native (tripolar) grid.  

The proposal is a two-step process.:
Step 1 is to use this interpolation code (`ocnicepost.fd`) to apply interpolation to the model output on the native grid to generate intermediate interpolated netCDF files.  
Step 2 is to use these intermediate interpolated netCDF files as input for GRIB2 product generation (wgrib2).

Does that make sense?

Reply:
@aerorahul Ice history output is specified in ice_in. 

I understand your step1,2. The question (I think) is whether there is any reason to limit the variables saved in the "intermediate interpolated netcdf".

Reply:
Not to my knowledge.  But based on the comments from @GwenChen-NOAA, it seems there is.  Otherwise, this PR should only be adding missing variables and not removing any variable from the CSV

Reply:
In this [document](https://docs.google.com/document/d/1AwYbHArLJ6KvD0HW5bUbNOiFJaVAWgWEmnMQ2g076TA/edit), model output tripolar netCDF files are referred as "history" files and will be saved under /com/gfs/.../**model_data**/ice/**history**. "Operational" tripolar netCDF and latlon GRIB2 files are products that will be located under /com/gfs/.../**products**/ice and distributed through NOMADS at https://nomads.ncep.noaa.gov/. Because NOMADS has limited bandwidth, we can only distribute selected variables (listed in the [document](https://docs.google.com/document/d/1AwYbHArLJ6KvD0HW5bUbNOiFJaVAWgWEmnMQ2g076TA/edit)) to the public.

Rahul, your proposal will need a separate step to generate operational tripolar netCDF files.

Reply:
If "operational tripole netcdf" files are required, then why can't they be created with a simple extraction command (eg NCO would be ``ncks -v list-of-variables in.nc out.nc``). I'm sure cdo has something similar. 

I don't really see what files distributed as tripole grid has to do w/ the interpolation from tripole to lat lon.

Reply:
@GwenChen-NOAA - Can I confirm that the need for  "Operational" tripolar netCDF was for GDAS, not GFS?

Reply:
@JessicaMeixner-NOAA, operational tripolar netCDF files are requested for GFSv17/GEFSv13 ocean and ice components. GDAS only requested background and analysis files.

Reply:
@GwenChen-NOAA  okay, I'm starting to remember some requests for that for ice, in particular.   I agree with @DeniseWorthen that if tripolar grid netcdf files are retained to be distributed over nomads and that we have more variables in the history file, that creating sub-set field could be done by ncks and/or other program such as cdo and then we could still have an intermediate step that maintained these additional variables in the intermediate netcdf lat-lon file. Is there any issue with this other than that it would create an additional step but has the advantage of allowing for maximum utility/variable output in the intermediate step?

Reply:
No, I'm fine with that. Like I said in my previous answer:

>It depends on the steps you do to generate operational tripolar netCDF files.
>If you first generate operational tripolar netCDF files from model outputs, then apply the interpolation algorithm, sst_h won't be in the operational netCDF files.
>If you apply the interpolation directly to model output tripolar netCDF files and generate GRIB2 files from the intermediate interpolated netCDF files, then those variables can be kept in the intermediate files. This approach requires operational tripolar netCDF files to be generated in a separate step.

It's up to you to decide which option to do. I'm fine either way.

Reply:
Apologies - I missed that @GwenChen-NOAA  thanks for reposting.  

It sounds like having the operational tripolar grid update being done as a separate step is the preferred option.

Reply:
I have updated the csv files to addition only, so the ocean.csv file is reversed back to the original and `sst_h` is added back to the ice.csv file.

@aerorahul and @DeniseWorthen, please review and approve the PR if it looks good to you. Thanks!

Reply:
I think this is fine. I believe it means that the netcdf lat/lon files will contain the same variable list that were present in the prototype runs. Can @LydiaStefanova-NOAA confirm?

Reply:
@jiandewang should also take a look, since he and @LydiaStefanova-NOAA will generate the netCDF and GRIB2 files for HR3 and HR4. Both of them were involved in planning the [GFSv17/GEFSv13 operational forecast files](https://docs.google.com/document/d/1AwYbHArLJ6KvD0HW5bUbNOiFJaVAWgWEmnMQ2g076TA/edit?pli=1).

Reply:
I am going to merge this PR with the added variables.  If there is more discussion to be had, please open a new issue or a discussion topic on these variables.